container_files/public_html/doc/builtin/sql/ValueExpression.md

```diff
@@ -649,13 +649,21 @@ The SQL function `jseval` allows for the inline definition of functions using Ja
     any SQL expressions and will be bound to the parameters passed in to the
     function.
 
+There are two ways that values in arguments can be represented in Javascript:
+simplified (the default), and a non-simplified representation that is accessed
+by adding a `!` character to the parameter name argument (for example, `!x,y`
+instead of `x,y`).
```

Comment:
The goal was to avoid breaking existing uses of jseval, without cluttering up the syntax.  Should we use `jseval2`?  Or `jsexec`?  Any other thoughts?  Because it's more complex to write a js function with ExpressionValues as inputs, I didn't want to make this the default way of doing things.